googlemaps.py

```diff
@@ -0,0 +1,59 @@
+import requests
+
+
+def input_locations():
+    """
+    This allows the user to input the locations of the origin and destination and then puts these into the function to
+    get the distance
+
+    :return: inputted origins and destination
+    """
+    origin = input("Your location: ")
+    destination = input("Your destination: ")
+    return origin, destination
+
+
+def get_distance(origin, destination):
+    """
+    This function takes the inputted origin and destination, then uses the google maps API to calculate the distances
+    and duration (take out if not needed) of the route. These are outputted for each mode of transport so they are all
+    available.
+
+    :param origin: inputted origin
```

Comment:
You can also add the data type if you wanted to make it more clear. The data type will then appear next to the parameter when you hover over the function name in the IDE. e.g.

`:type origin: string`

Example from my hw this week:
<img width="397" alt="Screenshot 2021-08-06 at 10 01 04" src="https://user-images.githubusercontent.com/56518485/128485939-20a39f2b-6d1d-4f63-8576-6f4e14502fee.png">

Reply:
Nice to know that, learned a lot :D updated!